### ISSUE TITLE
midi mode support

### ISSUE THREAD

**Issue by koriwi** [koriwi/freedeck-ino#6]

**Comment by legomannetje** [comment]
That can be done using MIDI over USB. Have done a school project with it!
Arduino code wise it won't be that difficult (It was done over serial)

**Comment by koriwi** [comment]
yes, the HID-Project library we use, is already capable of that.
